tests/parametric/test_otel_logs.py

```diff
@@ -0,0 +1,631 @@
+        )
+
+        requests = test_agent.requests()
+        logs_request = [r for r in requests if r["url"].endswith("/v1/logs")]
+        assert logs_request, f"Expected logs request, got {requests}"
+        assert logs_request[0]["headers"].get("api-key") == "key", f"Expected api-key, got {logs_request[0]['headers']}"
+        assert (
+            logs_request[0]["headers"].get("other-config-value") == "value"
+        ), f"Expected other-config-value, got {logs_request[0]['headers']}"
+
+
+@features.otel_logs_enabled
+@scenarios.parametric
+class Test_FR09_Log_Injection:
+    """FR09: Log Injection Tests"""
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_LOGS_OTEL_ENABLED": "true",
+                "DD_SERVICE": "testservice",
+                "DD_ENV": "testenv",
+                "DD_VERSION": "1.0.0",
+                "DD_TRACE_DEBUG": None,
+            },
+        ],
+    )
+    def test_log_injection_disabled_when_otel_enabled(self, test_agent, test_library, library_env):
+        """Log injection is disabled when OpenTelemetry Logs support is enabled."""
+        with test_library as library, library.dd_start_span("test_span") as span:
+            library.write_log(
+                "test_log_injection_disabled_when_otel_enabled", LogLevel.INFO, "test_logger", span_id=span.span_id
+            )
+
+        log_payloads = test_agent.wait_for_num_log_payloads(1)
+        log_record = find_log_record(log_payloads, "test_logger", "test_log_injection_disabled_when_otel_enabled")
+        resource = find_resource(log_payloads, "test_logger", "test_log_injection_disabled_when_otel_enabled")
+
+        # Verify trace correlation works
+        assert log_record.get("span_id") is not None
+        assert log_record.get("trace_id") is not None
+
+        # Verify service/env/version are ONLY in resource attributes
+        resource_attrs = find_attributes(resource)
+        assert resource_attrs.get("service.name") == "testservice"
+        assert resource_attrs.get("deployment.environment") == "testenv"
+        assert resource_attrs.get("service.version") == "1.0.0"
+
+        # Verify no duplication in log record attributes
+        log_attrs = find_attributes(log_record)
+        for dd_attr in ("service", "env", "version", "span_id", "trace_id"):
+            for log_attr in log_attrs:
+                assert (
+                    dd_attr not in log_attr
+                ), f"Found {dd_attr} in log attributes: {log_attrs}, should not duplicate resource attributes"
+
+
+@features.otel_logs_enabled
+@scenarios.parametric
+class Test_FR10_Timeout_Configuration:
+    """FR10: Timeout Configuration Tests"""
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {"DD_LOGS_OTEL_ENABLED": "true", "DD_TRACE_DEBUG": None, "DD_TELEMETRY_HEARTBEAT_INTERVAL": "0.1"},
+        ],
+    )
+    def test_default_timeout(self, test_agent, test_library, library_env):
```

Comment:
Since it's hard to test timeouts, this test feels like it's only a telemetry test. Should we just include this in the Telemetry set of tests instead?